README.md

```diff
@@ -593,6 +593,18 @@ WSDL.open('./wsdls/stockquote.wsdl',options,
 });
 ```
 
+### wsdl.openSync(wsdlURL, options)
+
+Loads WSDL into a tree form directly from memory. Traverse through WSDL tree to get to bindings, services, ports, operations, and so on as long as you have your dependant WSDLs and schemas are loaded and available in the `options.WSDL_CACHE`. If any I/O is required to retrieve any dependancies this call will throw an error.
```

Comment:
nitpicks/typos: `Traverse through` -> `It traverses through`,  `dependant` -> `dependent`, `dependancies` -> `dependencies`